resources/kcp/charts/kyma-environment-broker/files/swagger.yaml

```
@@ -2102,6 +2238,22 @@ components:
           type: string
         mount_config:
           $ref: '#/components/schemas/Object'
+ 
+    ServiceManagerCredentialsBasic:
+        username:
+          type: string
+        password:
+          type: string
+
+    ServiceManagerCredentials:
+      basic:
+        $ref: '#/components/schemas/ServiceManagerCredentialsBasic'
+
+    SMPlatformCredentials:
+      credentials:
```

`SMPlatformCredentials` should be typ `object` to render it's children properly

```suggestion
    SMPlatformCredentials:
      type: object
      credentials:
```